src/shared/components/navbar.tsx

```diff
@@ -5,48 +5,64 @@ import { i18n, LANGUAGES } from "../i18next";
 import classNames from "classnames";
 import { SELECT_CLASSES } from "./common";
 
-const NavLink = ({ content }) => <li className="text-gray-400">{content}</li>;
+const NAV_LINK = ({ content }) => (
+  <li className="text-gray-400 w-full md:w-fit">{content}</li>
+);
+
+const LINK_CLASS = "text-xl px-4  py-2 md:text-lg";
```

Comment:
Might be two spaces in between px-4 and py-2

Reply:
Fixed both things.